jax/lax/lax.py

```diff
@@ -4836,6 +4847,20 @@ def _cumprod_jvp_rule(primals, tangents, *, axis: int):
   return api.jvp(partial(_cumprod_prefix_scan, axis=axis), primals, tangents)
 
 
+def _cummax_jvp_rule(primals, tangents, *, axis: int, unit: Number):
```

Comment:
Can we share the implementation here and make it parametric?

Reply:
Done.